CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-12620: We continue to simplify access control management by grouping some permissions in permission sets. As a result:
   - The permission `WorkflowAdministration` will deprecate the permissions `Policy, VulnerabilityReports`.
 
+- ROX-14400: `BuildDate` is deprecated and will be removed in future releases. It won't be returned by
```

Comment:
1. Normally, we should announce in which specific release the thing will be removed.
2. Which ticket will be used for the actual removal?

Reply:
1. 3.75 should be OK
2. ROX-14336

Reply:
I'm not sure how used it is by users, but we typically have a two release deprecation cycle. Looking at how minor this is though (it's somewhat tied to `debug`), I'm fine with 3.75.
We've added the `Required Actions` section to be more clear about remediations that may require user interactions, for this I don't really think it makes too much sense to add something in that section, WDYT?

Reply:
> I don't really think it makes too much sense to add something in that section

I agree this could be an overkill. Anyway if anybody is using this I'm curious why.

Reply:
> I'm not sure how used it is by users, but we typically have a two release deprecation cycle.

@dhaus67 Do you know if this is documented somewhere? If not, we should document it. My first thought is to put it in the beginning of https://github.com/stackrox/stackrox/blob/master/CHANGELOG.md